lmfdb/users/main.py

```diff
@@ -51,11 +51,12 @@ def get_username(uid):
 @app.context_processor
 def ctx_proc_userdata():
     userdata = {}
-    userdata['userid'] = 'anon' if current_user.is_anonymous else current_user._uid
-    userdata['username'] = 'Anonymous' if current_user.is_anonymous else current_user.name
-    userdata['user_is_authenticated'] = current_user.is_authenticated
-    userdata['user_is_admin'] = current_user.is_admin
+    userdata['userid'] = 'anon' if current_user.is_anonymous() else current_user._uid
+    userdata['username'] = 'Anonymous' if current_user.is_anonymous() else current_user.name
+    userdata['user_is_authenticated'] = current_user.is_authenticated()
+    userdata['user_is_admin'] = current_user.is_admin()
     userdata['get_username'] = get_username
+    print "user data", userdata
```

Comment:
did the api change? bad :-1: 
also, the print statement in the last line should go away